src/constants.js

```diff
@@ -1,448 +1,10 @@
-// THIS FILE IS GENERATED, DO NO EDIT MANUALLY
-// For more information see the README.md
-/* eslint-disable dot-notation */
 'use strict'
-module.exports = Object.freeze({
 
-  // serialization
-  PROTOBUF: 0x50,
-  CBOR: 0x51,
-  RLP: 0x60,
-  BENCODE: 0x63,
-  JSON: 0x0200,
-  MESSAGEPACK: 0x0201,
+const table = require('./base-table')
+const constants = {}
 
-  // multiformat
-  MULTICODEC: 0x30,
-  MULTIHASH: 0x31,
-  MULTIADDR: 0x32,
-  MULTIBASE: 0x33,
+for (const [k, v] of Object.entries(table)) {
```

Comment:
As in the other file more meaning variable names might make sense e.g. (`[name, code]`).